versions/3.0.md

```diff
@@ -2826,188 +2826,329 @@ example:
+            ]
+          }
+        ]
+      },
+      "Dog": {
+        "description": "A representation of a dog. Note that `Dog` will be used as the discriminator value.",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Pet"
           },
-          "required": [
-            "packSize"
-          ]
-        }
-      ]
+          {
+            "type": "object",
+            "properties": {
+              "packSize": {
+                "type": "integer",
+                "format": "int32",
+                "description": "the size of the pack the dog is from",
+                "default": 0,
+                "minimum": 0
+              }
+            },
+            "required": [
+              "packSize"
+            ]
+          }
+        ]
+      }
     }
   }
 }
 ```
 
 ```yaml
-schemas:
-  Pet:
-    type: object
-    discriminator: petType
-    properties:
-      name:
-        type: string
-      petType:
-        type: string
-    required:
-    - name
-    - petType
-  Cat:  ## "Cat" will be used as the discriminator value
-    description: A representation of a cat
-    allOf:
-    - $ref: '#/components/schemas/Pet'
-    - type: object
+components:
+  schemas:
+    Pet:
+      type: object
+      discriminator:
+        propertyName: petType
       properties:
-        huntingSkill:
+        name:
+          type: string
+        petType:
           type: string
-          description: The measured skill for hunting
-          enum:
-          - clueless
-          - lazy
-          - adventurous
-          - aggressive
       required:
-      - huntingSkill
-  Dog:  ## "Dog" will be used as the discriminator value
-    description: A representation of a dog
-    allOf:
-    - $ref: '#/components/schemas/Pet'
-    - type: object
-      properties:
-        packSize:
-          type: integer
-          format: int32
-          description: the size of the pack the dog is from
-          minimum: 0
+      - name
+      - petType
+    Cat:  ## "Cat" will be used as the discriminator value
+      description: A representation of a cat
+      allOf:
+      - $ref: '#/components/schemas/Pet'
+      - type: object
+        properties:
+          huntingSkill:
+            type: string
+            description: The measured skill for hunting
+            enum:
+            - clueless
+            - lazy
+            - adventurous
+            - aggressive
+        required:
+        - huntingSkill
+    Dog:  ## "Dog" will be used as the discriminator value
+      description: A representation of a dog
+      allOf:
+      - $ref: '#/components/schemas/Pet'
+      - type: object
+        properties:
+          packSize:
+            type: integer
+            format: int32
+            description: the size of the pack the dog is from
+            default: 0
+            minimum: 0
+        required:
+        - packSize
+```
+
+#### <a name="discriminatorObject"></a>Discriminator Object
```

Comment:
@DavidBiesack you are correct.  Using `title` is a bad idea, and I've disallowed _inline_ schemas for consideration with the `discriminator`.